src/index.ts

```diff
@@ -35,5 +35,6 @@ export {
   CloudinaryImage,
   CloudinaryVideo,
   CloudinaryFile,
+  CloudinarySDK,
```

Comment:
twice?

Reply:
one is a default export. 

import {CloudinarySDK} from '...'
import CloudinarySDK from ...